connector/src/main/java/org/jboss/as/connector/deployers/ra/processors/CachedConnectionManagerSetupProcessor.java

```diff
@@ -90,6 +92,10 @@ private CachedConnectionManagerSetupAction(final ServiceName serviceName) {
         @Override
         public void setup(final Map<String, Object> properties) {
             try {
+                final CachedConnectionManager noTxCcm = noTxCcmValue.getOptionalValue();
```

Comment:
That part looks ok to me. Each of the CCM instances needs to operate in the same context, and the CCM doesn't know anything about possible deployments that are going to use it